source/reference/api/start.txt

```diff
@@ -91,6 +91,25 @@ Request Body Parameters
      - Required
      - Name of the destination cluster.
 
+   * - ``buildIndexes``
+     - boolean
+     - Optional
+     - Configures index builds during sync.
+
+       Supported Options:
+
+       * ``beforeDataCopy`` (the default) causes ``mongosync`` to build indexes
```

Comment:
Typically use - to indicate nested bullets, rather than *.

Reply:
Fixed